tests/multiple_occurrences.rs

```diff
@@ -128,3 +128,52 @@ fn multiple_occurrences_of_after_env() {
     assert!(m.is_ok(), "{}", m.unwrap_err());
     assert_eq!(m.unwrap().occurrences_of("verbose"), 3);
 }
+
+#[test]
+fn max_occurrences_implies_multiple_occurrences() {
+    let app = App::new("prog").arg(
+        Arg::new("verbose")
+            .short('v')
+            .long("verbose")
+            .max_occurrences(3),
+    );
+    let m = app.try_get_matches_from(vec!["prog", "-vvv"]);
+
+    assert!(m.is_ok(), "{}", m.unwrap_err());
+    assert_eq!(m.unwrap().occurrences_of("verbose"), 3);
+}
+
+#[test]
+fn max_occurrences_try_inputs() {
```

Comment:
Can you please check the error kind in this test case instead of `too many v's`.